extensions/amp-analytics/0.1/visibility-model.js

```diff
@@ -348,9 +353,17 @@ export class VisibilityModel {
   isVisibilityMatch_(visibility) {
     dev().assert(visibility >= 0 && visibility <= 1,
         'invalid visibility value: %s', visibility);
+    // Special case: If visiblePercentageMin is 100%, then it doesn't make
+    // sense to do the usual (min, max] since that would never be true.
```

Comment:
Is it (min, max] or [max, min)?

Reply:
Hmm...according to https://github.com/ampproject/amphtml/blob/master/extensions/amp-analytics/amp-analytics.md it's [min, max):
"Note that the lower bound (visiblePercentageMin) is inclusive while the upper bound (visiblePercentageMax) is not."
But lines 367, 368 below implement (min, max] :(
So, comment agrees with code, but these do not agree with doc. Which should we change? (Regardless of answer, I favor changing in separate PR.)

Reply:
nice catch, doc needs to be fixed.

Reply:
Thanks. Will do.

Reply:
Addressed in #12451